### PR TITLE
Convert transferPositionAndCall to a more structured transferPositionAndNotify (for hardhat)

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

File: test/liquidity-protection/LiquidityProtection.js [hunk 1]
```diff
@@ -2297,90 +2297,48 @@ describe('LiquidityProtection', () => {
...
-                            let testCall;
+                            let callback;
 
                             beforeEach(async () => {
-                                testCall = await Contracts.TestCall.deploy();
+                                callback = await Contracts.TestTransferPositionCallback.deploy();
                             });
 
-                            it('should revert when called with invalid call data', async () => {
+                            it('should revert when called with an invalid target', async () => {
```
Comment (barakman) [inline test/liquidity-protection/LiquidityProtection.js:2306]: So why call it 'callback' in one place and 'target' in another?

**Review by barakman — CHANGES_REQUESTED** [review]

**Comment by sshmaxime** [comment]
Seems like you are putting back the test and contracts files into the old path `soldity/test/` or `solidity/contracts`

**Comment by lbeder** [comment]
I’ll rename it here as well.

**Comment by lbeder** [comment]
Shit, it’s a result of merging master. I’ll fix it.

File: contracts/liquidity-protection/LiquidityProtection.sol [hunk 2]
```diff
@@ -773,23 +774,20 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
...
      */
-    function transferPositionAndCall(
+    function transferPositionAndNotify(
         uint256 id,
         address newProvider,
-        address target,
-        bytes memory data
-    ) external nonReentrant validAddress(newProvider) validAddress(target) returns (uint256) {
-        // make sure that we're not trying to call into the zero address or a fallback function
-        require(data.length >= FUNC_SELECTOR_LENGTH, "ERR_INVALID_CALL_DATA");
```
Comment (barakman) [inline contracts/liquidity-protection/LiquidityProtection.sol:788]: Should also remove the declaration of the `FUNC_SELECTOR_LENGTH` constant, wherever it is.

**Review by barakman — CHANGES_REQUESTED** [review]

File: contracts/liquidity-protection/LiquidityProtection.sol (hunk 2 above)
Comment (lbeder) [reply contracts/liquidity-protection/LiquidityProtection.sol:788]: Oh, right! It's at the beginning of the file, with the rest of the constants.

**Review by barakman — APPROVED** [review]

**Review by yudilevi — APPROVED** [review]

**Pull request merged**